experiments/build-swiftshader.md

```diff
@@ -0,0 +1,22 @@
+# Building SwiftShader with Coverage Enabled
+
+The SwiftShader CMakeLists.txt file defines the SWIFTSHADER\_EMIT\_COVERAGE flag to generate coverage information. However, it was designed for a very specific toolchain (clang-10 toolchain, CMake option SWIFTSHADER\_LLVM\_VERSION=10.0).
+
+The build at https://swiftshader-review.googlesource.com/c/SwiftShader/+/44728 can be built with coverage information without these restrictions.
+
+## Steps:
+
+1. Clone the [repo](https://swiftshader-review.googlesource.com/c/SwiftShader/+/44728)
```

Comment:
You can build a more recent version of SwiftShader. I suggest picking a recent commit hash e.g.
b6e8c3f0f4830887d69ba765a922ac3c40e81dd9

and cloning that.